content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty, it is possible that your browser cookie
+      settings prevented it from loading.
+      Note that these settings also change if you are in "Incognito" mode on your
+      browser.
+
+      To fix this issue, configure your browser to always allow cookies on Odoo's plugin page.
+
+      For Google Chrome, you can do so by following the guide at:
+      `https://support.google.com/chrome/answer/95647 <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
+      and adding `download.odoo.com` to the list of `Sites that can always use cookies`.
+
+      .. image:: outlook/cookie-fix.png
+         :align: center
+         :alt: Chrome cookie configuration to fix the empty panel issue
```

Comment:
Let's delete this image. For platforms outside of Odoo, we want to link to guides from those platforms, rather than walking through the instructions on Odoo's documentation. So what you did here by linking to Google's support page is perfect, and we don't need to include more Google Chrome content in addition to that.

Also, we try not to use too many images. Although your image is helpful, it is not completely necessary for the reader to understand the instructions.